.azure/templates/steps/setup_java.yaml

```diff
@@ -1,23 +1,23 @@
 # Step to setup JAVA on the agent
 parameters:
   - name: JDK_PATH
-    default: '/usr/lib/jvm/java-11-openjdk-amd64'
+    default: '/usr/lib/jvm/java-17-openjdk-amd64'
   - name: JDK_VERSION
-    default: '11'
+    default: '17'
 
 steps:
   - bash: |
       sudo apt-get update
     displayName: 'Update package list'
   - bash: |
-      sudo apt-get install openjdk-11-jdk
-    displayName: 'Install openjdk11'
-    condition: eq(variables['JDK_VERSION'], '11')
+      sudo apt-get install openjdk-17-jdk
+    displayName: 'Install openjdk'
+    condition: eq(variables['JDK_VERSION'], '17')
```

Comment:
Installing Java is often an issue in the CI pipelines. We should use the preinstalled Java -> you can check for example this PR as an inspiration: https://github.com/strimzi/drain-cleaner/pull/72/files

Reply:
Makes sense. let me check and update